schedule/functional/extra_tests_gnome.yaml

```diff
@@ -0,0 +1,10 @@
+name:   extra_tests_gnome
+description:    >
+    Maintainer: asmorodskyi, okurz.
+    Extra tests which were designed to run on gnome , VNC_STALL_THRESHOLD is needed for xen svirt to don't turn off the scrreen after default 4 sec
+schedule:
```

Comment:
I suppose that you will add the missing test modules? https://openqa.suse.de/tests/3796732